app/src/main/java/org/kiwix/kiwixmobile/zim_manager/library_view/LibraryPresenter.java

```diff
@@ -33,6 +34,7 @@ void loadBooks() {
         .subscribe(library -> {
           getMvpView().showBooks(library.getBooks());
         }, error -> {
+          Log.w("kiwixLibrary", error.getLocalizedMessage());
```

Comment:
Thanks for letting me know. I've added some guard code. When I tested it on Amtrak (which provides a slow, sporadic network connection) the call errored and the error message was 'timeout'. I hope the next time the code is run on the TestDroid devices it'll work without crashing.